Remove Kickoff traces (#313)
SORSE/sorse.github.io: _posts/programme/2020-08-19-kickoff.md

meeting_url: https://zoom.us/j/95982897830

> last_modified_at: 2020-09-02

Reviewer: ```suggestion
last_modified_at: 2020-09-04
```

---

> To kick off our series off, we have run the SORSE Launch

Reviewer: ```suggestion
To kick off our series, we ran the SORSE Launch
```

Author: Ah, nice catch, the duplicate off. Thanks.

I prefer perfect in this case as it it something that reverberates into the present, but either way is fine.

Reviewer: As you like. I just used simple past because you used it down below, too.

Author: Nah, it's fine :).

Event on Wednesday 2nd September. We had
two keynote speakers at this event:

- Dr Mariann Hardey, Associate Professor, Durham University: _Switching off the label "women in tech"_
- Dr Kari Jordan, Executive Director, The Carpentries: _I wanna dance with somebody_

The session began with a short welcome and introduction to SORSE followed

> by two 30 minute keynote talks. Dr Mariann Hardey spoke on the subject of
> women in tech and Dr Kari Jordan spoke about the Carpentries. The talks were

Reviewer 2: ```suggestion
by two 30 minute keynote talks. Dr Kari Jordan spoke about the Carpentries 
and Dr Mariann Hardey spoke on the subject of women in tech. The talks were
```

Author: Have thought about the order while working on this, but was pressed for time, so didn't change. Nice one!

followed by time for questions and then an opportunity for further discussion
and networking in breakout rooms.

> <i class="fab fa-fw fa-youtube"></i> A recording of this event is [available on YouTube](https://www.youtube.com/watch?v=2HOznlOCd1w).

Reviewer 2: What about also adding the video directly into the site? Should be easy through the [youtube helper](https://mmistakes.github.io/minimal-mistakes/docs/helpers/#youtube) provided by our theme.

Author: Sure!
